funcbench/Makefile

```diff
@@ -1,24 +1,24 @@
 INFRA_CMD        ?= ../infra/infra
 
 .PHONY: deploy clean
-deploy: nodepool_create resource_apply
+deploy: cluster_create resource_apply
 # GCP sometimes takes longer than 30 tries when trying to delete nodepools
 # if k8s resources are not already cleared
-clean: resource_delete nodepool_delete
+clean: resource_delete cluster_delete
 
-nodepool_create:
-	$(INFRA_CMD) gke nodepool create -a ${AUTH_FILE} \
-		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
-		-f manifests/nodepools.yaml
+cluster_create:
+	$(INFRA_CMD) gke cluster create -a ${AUTH_FILE} \
+		-v PROJECT_ID:${PROJECT_ID} -v ZONE:${ZONE} -v CLUSTER_NAME:funcbench-${PR_NUMBER} \
```

Comment:
Seems variable `CLUSTER_NAME` defined in [workflow file](https://github.com/prometheus/prometheus/blob/master/.github/workflows/funcbench.yml#L12) is useless now.

Reply:
Thanks, will update.